packages/bindings/binding.gyp

```diff
@@ -62,7 +65,26 @@
             'src/poller.cpp'
           ]
         }
-      ]
-    ]
+      ],
+
+      # Coverage generation options
+      # These coverage options are currently only compatible with non-Windows OSes
+      ['generate_coverage=="yes"',
+        {
+          'cflags+': ['--coverage'],
+          'link_settings': {'libraries+': ['-lgcov']},
+          'xcode_settings': {
```

Comment:
I've seen others put macOS conditions around `xcode_settings` but that doesn't seem to be necessary as far as I've seen.